app/controllers/sections_controller.rb

```diff
@@ -230,6 +230,21 @@ def uncrosslist
   # @API Edit a section
   # Modify an existing section.  See the documentation for {api:SectionsController#create create API action}.
```

Comment:
since the parameters are copied now, there's no longer any need to refer to the Create documentation